src/geometry.cpp

```diff
@@ -490,4 +491,62 @@ extern "C" int openmc_global_bounding_box(double* llc, double* urc)
   return 0;
 }
 
+extern "C" double openmc_calculate_optical_thickness(
+  Position start_sampled_position, Position end_sampled_position)
```

Comment:
Not only does this match your declaration in the header, there may be use cases for this where the positions aren't "sampled"
```suggestion
  Position start_pos, Position end_pos)
```
(and rename all subsequent uses)

Reply:
hmm. I wonder why it still runs properly with the header discrepancy. I wonder if it only considers the types and method name rather than the parameter names